index.html

```diff
@@ -1,453 +1,585 @@
-          normalization.
-        </dd>
-        <dt>
-          <dfn data-lt="message digest algorithm | message digest algorithms"
-            >message digest algorithm</dfn
-          >
-        </dt>
-        <dd>
-          An algorithm that takes a message, prefferably in some
-          <a>canonical form</a> and produces a cryptographic output called a
-          digest that is often many orders of magnitude smaller than the input
-          message. These algorithms are often 1) very fast, 2) non-reversible,
-          3) cause the output to change significantly when even one bit of the
-          input message changes, and 4) make it infeasible to find two different
-          inputs for the same output.
-        </dd>
-        <dt><dfn>canonical form</dfn></dt>
-        <dd>
-          The output of applying a <a>canonicalization algorithm</a> to an input
-          document.
-        </dd>
-        <dt><dfn>signature algorithm</dfn></dt>
-        <dd>
-          An algorithm that takes an input message and produces an output value
-          where the receiver of the message can mathematically verify that the
-          message has not been modified in transit and came from someone
-          possessing a particular secret.
-        </dd>
-        <dt>
-          <dfn data-lt="linked data document|linked data documents">linked data document</dfn
-          >
-        </dt>
-        <dd>A document comprised of linked data.</dd>
-        <dt>
-          <dfn data-lt="linked data proof|linked data proofs">linked data proof</dfn>
-        </dt>
-
-        <dt><dfn>EcdsaSecp256k1VerificationKey2019</dfn></dt>
-        <dd>
-          A <code>type</code> of the verification method for the signature
-          suite <a>EcdsaSecp256k1VerificationKey2019</a>.
-        </dd>
-
-        <dt><dfn>EcdsaSecp256k1RecoveryMethod2020</dfn></dt>
-        <dd>
-          A <code>type</code> of the verification method for the signature
-          suite <a>EcdsaSecp256k1RecoveryMethod2020</a>.
-        </dd>
-
-        <dt><dfn>EthereumEip712Signature2021</dfn></dt>
-        <dd>
-          The <code>type</code> of the linked data proof for the signature suite
-          <a>EthereumEip712Signature2021</a>.
-        </dd>
-      </dl>
+      It will generate the following schema:
+      <pre class="example">
+          {
+            Name: [
+              { name: "first", type: "string" },
+              { name: "last", type: "string" },
+            ],
+            OtherData: [
+              { name: "jobTitle", type: "string" },
+              { name: "school", type: "string" },
+            ],
+            Document: [
+              { name: "@context", type: "string[]" },
+              { name: "@type", type: "string" },
+              { name: "name", type: "Name" },
+              { name: "otherData", type: "OtherData" },
+              { name: "telephone", type: "string" },
+              { name: "email", type: "string" },
```

Comment:
I compute that `email` comes before `name`:
```suggestion
              { name: "@type", type: "string" },
              { name: "email", type: "string" },
              { name: "name", type: "Name" },
              { name: "otherData", type: "OtherData" },
              { name: "telephone", type: "string" },
```

Sorting is also being discussed in https://github.com/w3c-ccg/ethereum-eip712-signature-2021-spec/pull/25/files#r702969162